src/modules/ethereum/transferTransactions.js

```diff
@@ -42,6 +42,28 @@ export function getMaxTransfer(amount) {
     return maxAmount;
 }
 
+export async function transferEthTx(payload) {
+    const { payee, ethAmount } = payload;
+
+    const gasEstimate = cost.TRANSFER_AUGMINT_TOKEN_GAS; // TODO MAYBE change to & check ETH_GAS?
```

Comment:
Indeed, that’s a very high estimate. Should use the default for transfers.

Reply:
shall we do it properly?
Add a new constant: `ETH_TRANSFER_GAS = 21000`

Reply:
Doesn't web3 have a sensible default when unspecified? (Though a constant certainly works).